dftimewolf/lib/containers/containers.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import annotations
 
 import datetime
+import pandas as pd
```

Comment:
Remove import from the TYPE_CHECKING block

Reply:
Done